packages/driver/src/cypress/error_messages.ts

```diff
@@ -1803,6 +1834,18 @@ export default {
   },
 
   subject: {
+    is_null (obj) {
+      return stripIndent`\
+        ${cmd(obj.name)} failed because it requires a subject, but didn't receive one. This usually happens because you're trying to use a child query as if it were a parent.`
```

Comment:
I think we should consider aligning this verbiage with https://github.com/cypress-io/cypress/blob/issue-7306-addSelector-redux/packages/driver/src/cypress/error_messages.ts#L874 ('oops, looked like ...') and provide pusedo code on how to fix this. If someone isn't as familiar with parent vs child command this likely won't make sense
```suggestion
        ${cmd(obj.name)} failed because it requires a subject, but it didn't receive one. This usually happens because you're trying to use a child query command as if it were a parent command`
```

Reply:
Since ensureNull  / ensureNotNull aren't actually used in this PR - and in fact I haven't found a good use for them yet - I went ahead and removed them entirely, along with the associated error messages. Leftover stuff from earlier iterations on this PR that I hadn't cleaned up.